docs/ocis/development/testing.md

```diff
@@ -37,11 +37,19 @@ You can invoke two types of test suite runs:
 
 ### Run full test suite
 
-The names of the full test suite make targets have the same naming as in the CI pipeline.
+The names of the full test suite make targets have the same naming as in the CI pipeline. The available local ocis specific test suites are `apiAccountsHashDifficulty`, `apiArchiver`, `apiContract`, `apiGraph`, `apiSpaces`, `apiSpacesShares`, `apiCors`, `apiAsyncUpload`. They can be run with `oCIS` storage and `S3` storage.
 
```

Comment:
Before running test we can add a test log section here instead of down to see our test running while we use the command.